collection/search_specifications_test.go

```diff
@@ -144,7 +144,7 @@ func ExampleCollection_SearchSpecifications() {
 
 	nc := collection.NewCollection(k)
 
-	res, err := nc.SearchSpecifications(nil)
+	res, err := nc.SearchSpecifications(json.RawMessage("{}"), nil)
```

Comment:
```suggestion
	res, err := nc.SearchSpecifications(json.RawMessage( `{}`), nil)
```
Nitpicking: But to stay consistent with the other lines :slightly_smiling_face: